### ISSUE TITLE
Membership ?user and ?collection filters don't filter anything

### ISSUE THREAD

**Issue by jonboiser** [learningequality/kolibri#3272]
When analyzing #3271, I noticed that the filters for the Membership API don't seem to filter anything out e.g.

```
/api/membership/?user=abcd123
```

returns all Memberships.

This is a problem because the `removeClassUser` action relies on this filter, so removing a single user removes all Memberships in the system.

On `learner-lessons`, I patched this particular issue with this change to `MembershipFilter`

```
class MembershipFilter(FilterSet):
    user_ids = CharFilter(method="filter_user_ids")

    def filter_user_ids(self, queryset, name, value):
        return queryset.filter(user_id__in=value.split(','))

    class Meta:
        model = Membership
        fields = ('user', 'collection', 'user_ids',)

```

**Comment by rtibbles** [comment]
Yeah, filters won't work unless the fields are explicitly listed, good catch, seems like the right fix.

**Comment by jonboiser** [comment]
Should be fixed in d5ff53953af41f41e675af612d1e1d7d56ad4f6b
